src/OSPSuite.Presentation/Formatters/MultiCurveOptionsFormatter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using OSPSuite.Assets;
+using OSPSuite.Core.Chart;
+using OSPSuite.Utility.Format;
+
+namespace OSPSuite.Presentation.Formatters
+{
+   public class BooleanYesNoFormatter : IFormatter<bool?>
+   {
+      public string Format(bool? valueToFormat)
+      {
+         if (valueToFormat == null)
+            return Captions.Chart.MultiCurveOptions.CurrentValue;
+
+         return  valueToFormat.Value ? "Yes" : "No";
```

Comment:
We have constant for this somewhere. They should be used

Reply:
@msevestre hmm... I have searched for "Yes" but there does not seem to be a constant for that. I changed the strings to ViewResult.Yes.ToString() and ViewResult.No.ToString(). Will that do, or should it be some other way?

Reply:
Create a string in the cosntnat that we have with "yes" and "no" 

      public static readonly string No = "No";
         public static readonly string Yes = "Yes";

Reply:
done